lib/modules/manager/npm/post-update/npm.ts

```diff
@@ -171,7 +171,7 @@ export async function generateLockFile(
     // because npm install was called with an explicit version for rangeStrategy=update-lockfile
     if (lockUpdates.length) {
       const { detectedIndent, lockFileParsed } = parseLockFile(lockFile);
-      if (lockFileParsed?.lockfileVersion === 2) {
+      if (lockFileParsed?.lockfileVersion >== 2) {
```

Comment:
so we need to add lockfile v3 support explicit? what's the difference?

Reply:
npm lockfile v3 also needs "massaging" like already implemented for npm lockfile v2